sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -287,9 +319,123 @@ Provide an Avalon memory mapped interface as described in https://www.intel.com/
 Provide an AXI4-Stream interface as described in https://documentation-service.arm.com/static/642583d7314e245d086bc8c9[AMBA 4 AXI4-Stream Protocol Specification].
 
 The default protocol is *avalon_streaming*
+
+|`latency_anchor_id<N>`
+| Valid values: Non-negative integer values of N
+
+This property associated an ID with the current read/write function call, which can then be referenced by other `latency_constraint` properties elsewhere in the program to define relative latency constraints. The ID must be unique within the application, and a diagnostic is required if that condition is not met.
+
+This property is only valid when passed as a function argument of a device side read/write method as a properties object.
+
+|`latency_constraint<A, B, C>`
+| Valid values: `A` and `C` are non-negative integer values, while `B` is a enum value from latency_control_type.
+
+This property is a tuple of three values which cause the current read/write function call to act as an endpoint of a latency constraint relative to a specified `latency_anchor_id` defined by a different instruction.
+
+`A` is the ID of the target anchor defined on a different instruction through a `latency_anchor_id` property.
+ 
+`B` is an enum value: The type of control from the set {`latency_control_type::exact`, `latency_control_type::max`, `latency_control_type::min`}.
+
+`C` is the relative clock cycle difference between the target anchor and the current function call that the constraint should infer subject to the type of the control (exact, min, max).
+
+This property is only valud when passed as a function argument of a device side read/write method as a properties object.
+
```

Comment:
I don't think it makes sense to put these in the same table as the other properties because they go on the read/write calls instead of the pipe type.

Reply:
I've broken up the table into two parts. One for properties that can be applied to the pipe type, and another for properties that can be applied to the reads/writes to the pipe.